mdp/nodes/expansion_nodes.py

```diff
@@ -358,14 +475,21 @@ def is_invertible():
     def pseudo_inverse(self, x, use_hint=None):
         """Calculate a pseudo inverse of the expansion using
         scipy.optimize.
-
-        ``use_hint``
-               when calculating a pseudo inverse of the expansion,
-               the hint determines the starting point for the approximation.
-               For details on this parameter see the function 
-               ``invert_exp_funcs2`` in ``mdp.utils.routines.py``.
-
-        This method requires scipy."""
+        
+        This method requires scipy.
+        
+        :param x: The data.
+        :type x: numpy.ndarray
+        
+        :param use_hint: When calculating a pseudo inverse of the expansion,
+            the hint determines the starting point for the approximation.
+            For details on this parameter see the function 
+            ``invert_exp_funcs2`` in ``mdp.utils.routines.py``.
+        :type: numpy.ndarray or bool
```

Comment:
Should this rather be ``:type use_hint:`` ?

Reply:
adapted it